swagger/api.yaml

```diff
@@ -138,6 +138,17 @@ paths:
         requestParameters:
           integration.request.path.proxy: "method.request.path.proxy"
 
+  # CMS API (Studio ingestion)
```

Comment:
The proposed “cms” path name represents a service, and [not a platform subdomain according to our docs](https://docs.openedx.org/projects/openedx-events/en/latest/how-tos/creating-new-events.html#place-your-event-in-an-architecture-subdomain). Should this be replaced with “content-authoring”? If an endpoint is needed for “content-authoring” that is covered by a different service in the future, will someone know where to add this endpoint?

All this being said, you are the owners and I don't want to be a blocker. I just want to provide a reminder about this thinking and ensure there is an eye to the future, given that the whole benefit of the gateway is to have some independence from the actual underlying services.

Reply:
I brought up your point regarding cms being a service and not a domain name at our biweekly API team meeting. We are OK changing the API name eventually, but making the change away from CMS now would require touching code and documentation in multiple places, and we'd prefer to push  back on doing that now. The API we seek to publish is still unfit to service pretty much anyone's business needs. Our goal is to empower sales to hold conversations that will inform us as to actual requirements for the API. As such, we see very limited use of the API for some time, and little penalty in making changes in say six months. In the meantime, there's more pressing work.

Reply:
I don't want to block you, but I'm also not clear on your response.

1. Once you publish and publicize an API, it can't easily be changed without potentially breaking existing usage. If you expect no usage, why do this at all? If you expect minor testing usage, you could consider `v0` or some other indicator that this may break.
2. Wouldn't changing the name just involve changing the paths from `"/cms/v1/file_assets/{proxy+}":` to `"/authoring/v1/file_assets/{proxy+}":`? Couldn't everything else stay the same?
3. Doesn't someone need to know that they could call `https://api.edx.org/authoring/*` to use these new endpoints, and thus some doc would need to be updated, whether it read `cms` or `authoring` in the name?

Reply:
Yes. In order, per your comments:
1) The public APIs are now versioned as v0
2) 'cms' tokens in the public endpoints are now 'authoring'
3) We'll need to make corresponding changes to our readthedocs document for the cms -> authoring change

Reply:
Thanks. Marking this resolved.